docs/BUILD

```diff
@@ -1 +1,9 @@
 python_sources()
+
+poetry_requirements(
```

Comment:
Hi @dwight-foster, 
Could you share some insights into why this change is required?

Reply:
It was not required. I was trying to figure out why the build failed the first time and so I tried a bunch of ways at the same time. This was what other build files in llm integrations looked like so I changed it to that.